### ISSUE TITLE
Keeping history even with keep_history: false

### ISSUE THREAD

**Issue by pyrrhicPachyderm** [crazy-max/ghaction-github-pages#27]
I may be misinterpretting what `keep_history: false` is supposed to do, but it doesn't seem to be doing it, even when I set is explicitly.

It is my understanding that it's supposed to get rid of any old commits on the target branch, so that the target branch only ever has one commit at a time. This is the desired behaviour for me, as my compiled material is quite large and will quickly take up a lot of space.

But my target branch is accumulating commits each time the action runs, as you can see [here](https://github.com/pyrrhicPachyderm/coven-rpg/tree/gh-pages) and [here](https://github.com/pyrrhicPachyderm/github-pages-deploy-test/tree/gh-pages). This is my configuration (identical for both repos):
```name: Deploy to GitHub Pages
if: success()
uses: crazy-max/ghaction-github-pages@v1
with:
  target_branch: gh-pages
  build_dir: website
  keep_history: false
env:
  GITHUB_PAT: ${{ secrets.GITHUB_PAT }}```

**Comment by crazy-max** [comment]
Hi @pyrrhicPachyderm,

You absolutely right, good catch!
Will be solved asap, keep you in touch!

**Comment by crazy-max** [comment]
Should be fixed with latest release (`v1`, `v1.2` or `v1.2.4` tag)

**Comment by pyrrhicPachyderm** [comment]
Awesome! Works perfectly! Thanks very much for the quick fix!
